sphinxodoo/ext/autodoc.py

```diff
@@ -29,8 +29,12 @@ def load_odoo_modules(addons):
             except ImportError:
                 # skip non module directories
                 continue
+            if use_odoo_namespace:
+                namespace = 'odoo'
+            else:
+                namespace = 'openerp'
             mod = imp.load_module(
-                'openerp.addons.%s' % module_name, f, path, descr)
+                '%s.addons.%s' % (namespace, module_name), f, path, descr)
```

Comment:
I tend to prefer the current version as I find it easier to understand. I already feel guilty about the aliasing of the module name, and I fear someone may think "oh this can obviously be simplified to 'openerp'"